Add solution for Components in a graph in C++

itish-agarwal opened the pull request:
Add solution in C++ in folder Disjoint Sets/Components in a graph for the problem Components in a graph

itish-agarwal committed:
Add solution for Components in a graph in C++

stormfireuttam reviewed (commented):
Please update the Readme as well.

itish-agarwal committed:
Update README.md 

Update README.md to include the problem Components in a Graph under the topic Disjoint Sets
